### PR TITLE
Update NuGet-FAQ.md

### PR DESCRIPTION
**Pull request by diverdan92**
Changing to account for package ID prefix reservation. Will point to new docs when the feature is advertised publicly soon

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/Policies/NuGet-FAQ.md](https://github.com/diverdan92/docs.microsoft.com-nuget/blob/patch-1/docs/Policies/NuGet-FAQ.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C9%5C20%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201709202120522484-480%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=lDHlfrXYxL4mBPGuenB6QhpE7eXbehC9N1%2FudGhIGog%3D&st=2017-09-20T21%3A16%3A39Z&se=2017-09-21T21%3A26%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/Policies/NuGet-FAQ.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ This is not an issue when using PackageReference, as each project file contains
 
 **Is it possible to reserve names for packages that will be published in future?**
 
-No. If you feel that an existing package has taken the name which suits your package better, try contacting the owner of the package (select on the owner name on the package page on nuget.org). If you don't get a response within a reasonable time, [contact support](https://www.nuget.org/policies/Contact) and we'll look into it.
+Yes. You can reserve IDs for packages on [nuget.org](https://www.nuget.org/) by requesting a package ID prefix for your account. In order to request a package ID prefix, send mail to account@nuget.org with the package owner display name, and the requested package ID prefix.
```
Comment (karann-msft) [inline docs/Policies/NuGet-FAQ.md]: Can we make the email link smart such that the subject line is prepopulated and some content is already in the body of the email and such.

Comment (diverdan92) [reply docs/Policies/NuGet-FAQ.md]: Do you know how to  do that?

**Comment by saldana** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/NuGet/docs.microsoft.com-nuget, working directory: W:\ov1y-s, target branch: live, target branch commit: 9ba96b79af036deb670cd04ee48cbd84c8e3f4c2, source branch: 0019b1c14db2c3accbdc5b5f6220b58e/patch-1, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201710241621126722-480%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Y24TVVTxe7hvgMjxrz3tSOuk%2Fe70r%2BdwE0TTWv9OrT4%3D&st=2017-10-24T16%3A16%3A22Z&se=2017-11-24T16%3A21%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/Policies/NuGet-FAQ.md](https://github.com/diverdan92/docs.microsoft.com-nuget/blob/patch-1/docs/Policies/NuGet-FAQ.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201710241622099910-480%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Rc9YT0vP7HjzHPkNCXje9mfpT%2BD7fUKu4aLAokOY81o%3D&st=2017-10-24T16%3A17%3A50Z&se=2017-11-24T16%3A22%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**